R/file_utils.R

```diff
@@ -666,15 +667,17 @@ load_twas_weights <- function(weight_db_files, conditions = NULL,
           FALSE
         }
       )
-    })]
+    })] 
     combined_susie_result <- lapply(conditions, function(condition) {
```

Comment:
i suspect this is no longer needed if you put `susie_weights_intermediate` in univriate and multivariate TWAS weights pipelines? Of course you only do that if susie_weight is specified.

Reply:
yes, this is now fixed